dht/cdc_partitioner.cc

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
Question on the commit message (because I can't put it anywhere else... github...):
Can you please explain better what's the idea of having two-integer partition keys, where only the first integer is the token? Until now I was under the impression that what we're trying to do is that the partition key - the stream id - will be the same as the token? What do we need that second integer for and if we do, why is it not part of the token?

Reply:
I will improve the commit message and add comments. Thanks.